checks.d/mysql.py

```diff
@@ -781,7 +790,7 @@ def _get_stats_from_variables(self, db):
 
     def _get_binary_log_stats(self, db):
         with closing(db.cursor()) as cursor:
-            cursor.execute("SHOW MASTER LOGS;")
+            cursor.execute("SHOW BINARY LOGS;")
```

Comment:
Shouldn't there be the same raise/except logic for this query, in cases when the user doesn't have `REPLICATION CLIENT` rights? (cf https://dev.mysql.com/doc/refman/5.6/en/show-binary-logs.html)